docs/tasks.md

```diff
@@ -142,49 +119,70 @@ spec:
       emptyDir: {}
 ```
 
-### Steps
+### `Task` vs. `ClusterTask`
 
-The `steps` field is required. You define one or more `steps` fields to define
-the body of a `Task`.
+A `ClusterTask` is a `Task` scoped to the entire cluster instead of a single namespace.
+A `ClusterTask` behaves identically to a `Task` and therefore everything in this document
+applies to both.
 
-If multiple `steps` are defined, they will be executed in the same order as they
-are defined, if the `Task` is invoked by a [`TaskRun`](taskruns.md).
-Each `steps` in a `Task` must specify a container image that adheres to the
-[container contract](./container-contract.md). For each of the `steps` fields,
-or container images that you define:
+**Note:** When using a `ClusterTask`, you must explicitly set the `kind` sub-field in the `taskRef` field to `ClusterTask`.
+          If not specified, the `kind` sub-field defaults to `Task.` 
 
-- The container images are run and evaluated in order, starting from the top of
-  the configuration file.
-- Each container image runs until completion or until the first failure is
-  detected.
-- The CPU, memory, and ephemeral storage resource requests will be set to zero
+Below is an example of a Pipeline declaration that uses a `ClusterTask`:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: demo-pipeline
+  namespace: default
+spec:
+  tasks:
+    - name: build-skaffold-web
+      taskRef:
+        name: build-push
+        kind: ClusterTask
+      params: ....
+```
+
+### Defining `Steps`
+
+A `Step` is a reference to a container image that executes a specific tool on a
+specific input and produces a specific output. To add `Steps` to a `Task` you
+define a `steps` field (required) containing a list of desired `Steps`. The order in
+which the `Steps` appear in this list is the order in which they will execute.
+
+The following requirements apply to each container image referenced in a `steps` field:
+
+- The container image must abide by the [container contract](./container-contract.md). 
```

Comment:
whaddaya think about the container contract doc @sergetron ? im wondering if in the long run we should just fold it into this doc, its so short and doesnt seem like keeping it on its own is doing all that much good

Reply:
I think we can fold the contract into this doc as an appendix so it's not in the way of the main reading and just link to it from the relevant sections.